config_parser_test.cc

```diff
@@ -9,3 +10,207 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+class NginxConfigParserStringTest : public ::testing::Test {
+    protected:
+        virtual bool parseString(std::string s) {
+            std::stringstream test_config_stream(s);
+
+            return parser_.Parse(&test_config_stream, &config_);
+        }
+
+        NginxConfigParser parser_;
+        NginxConfig config_;
+};
+
+TEST_F(NginxConfigParserStringTest, StatementParsed) {
+  bool success = parseString("port 8080;");
```

Comment:
While consistent, the need to create a boolean variable that stores the value of function call parseString is redundant.